packages/test/test-driver-definitions/package.json

```diff
@@ -79,10 +81,10 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"disabled": true,
-		"version": "2.0.0-internal.2.2.0",
-		"baselineRange": ">=2.0.0-internal.2.1.0 <2.0.0-internal.2.2.0",
-		"baselineVersion": "2.0.0-internal.2.1.0",
+		"version": "2.0.0-internal.3.2.0",
```

Comment:
Probably fine for now, though this package is on the list of "can we make it internal?" (in which case we'll be nuking the typetests entirely).  I think the only annoyance is that the azure packages use this one currently.

So no objection to this as is, but if you want to see whether this can be made internal that would be another direction to go :)